spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -391,6 +390,15 @@ to facilitate the apps binding to the RabbitMQ service. Assuming your RabbitMQ s
 ```
 cf set-env dataflow-server SPRING_APPLICATION_JSON '{"spring.cloud.dataflow.applicationProperties.stream.spring.rabbitmq.addresses": "${vcap.services.rabbit.credentials.protocols.amqp.uris}"}'
 ```
+For Spring Cloud Task apps, something similar to the following could be used, if using a database service instance named `mysql`:
+
+```
+cf set-env dataflow-server spring.cloud.dataflow.applicationProperties.task.spring.datasource.url '${vcap.services.mysql.credentials.jdbcUrl}'
```

Comment:
Hmmm, I get it now. The server passes those because of https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskService.java#L204